public/locales/en/common.json

```diff
@@ -210,7 +210,7 @@
   "Latest episode": "Latest episode",
   "left": "Left",
   "Learn more": "Learn more",
-  "License and free trial info": "All Podverse software is provided under a free and open source (FOSS) licence. Features that require updating our servers are available only with a Premium membership. Sign up today and get a 3 month free trial.",
+  "License and free trial info": "It is all copylefted libre software. Use, study, change and share — with all.\nFeatures that use our servers are available in Podverse Premium. The first three months is a gift to you.",
```

Comment:
Synergy with
https://github.com/podverse/podverse-rn/pull/1373/files#diff-434395122634f503f334976454c18e6e9a440d8416c11d3a921f36eef09e6088R750
  "You are not being charged during your free trial": "The gifted period is gratis, and it is always your (manual) choice to get or renew Podverse Premium."

and

https://github.com/podverse/podverse-web/pull/888/files#diff-4840952983d2a94f27deb00bf7247347345b60e3a96dc2aa41a6a50ecacdf1b6R359
"PremiumFreeTrial": "Premium — Gifted period",